### PR TITLE
fix handwired/swiftrax/cowfish PID and VID

### PR DESCRIPTION
**Pull request by swiftrax**
## Description

Change Vendor ID and Product ID 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/swiftrax/cowfish/config.h [hunk 1]
```diff
@@ -20,8 +20,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0x7161
-#define PRODUCT_ID      0x5239
+#define VENDOR_ID       0x04D8
```
Comment (drashna) [inline keyboards/handwired/swiftrax/cowfish/config.h:23]: This VID belongs to Microchip.

Comment (swiftrax) [reply keyboards/handwired/swiftrax/cowfish/config.h]: Thanks reverted it. The PID was given to me by them

Comment (tzarc) [reply keyboards/handwired/swiftrax/cowfish/config.h]: if they've granted you a PID, then aren't you allowed to use their VID?

Comment (swiftrax) [reply keyboards/handwired/swiftrax/cowfish/config.h:23]: image

Comment (tzarc) [reply keyboards/handwired/swiftrax/cowfish/config.h:23]: Perfect, cheers for that.

**Pull request merged**